### DISCUSSION TITLE
I can't find my school on the map when I apply for the Student Developer Pack, what should I do?

### DISCUSSION THREAD

**Discussion by Gtd232** [community/community#45858] [GitHub Education]
I can't find my school on the map when I apply for the Student Developer Pack.

I'm applying for the Student Developer Pack and it says I need to find my school on the map, but my school is new and doesn't show the name, and the search box below can’t find it.What should I do?

Translated with DeepL (https://www.deepl.com/app/?utm_source=ios&utm_medium=app&utm_campaign=share-translation)
![IMG_4675](https://user-images.githubusercontent.com/79702405/215658151-f029492c-0d24-4e16-b425-3a6f435128a2.jpeg)

**Comment by jge162** [comment]
You should reach out to [GitHub Education support](https://support.github.com/request/education?tags=hubberfy_education) and submit a request for help.

**Reply by Gtd232** [reply]
Thanks.
